### PR TITLE
dotCMS/core#25130 [UI] Add to the Goal list the coming soon goals

### LINKED ISSUE #25130: [UI] Add to the Goal list the `coming soon goals`
### Parent
#24343 

### User Story
As a user, I want to see the coming next goals we are working

### Acceptance Criteria
Add 4 Goals as coming soon shown in the UI:
- Click on Element
- Time on site 
- Time on Page 
- Rules-based
### Proposed Objective

Core Features

### Figma

https://www.figma.com/file/fAoCk4swhiwplkU5GP21YD/Portlet-%2F-Experiments?type=design&node-id=1816-13266&t=Fr11ygR0L2UPaGMp-0

### Proposed Priority

Priority 2 - Important

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

Screenshot 2023-06-14 at 3 51 55 PM

### PR DESCRIPTION
**Pull request by hmoreras**
### Proposed Changes
* Add a placeholder for the coming goals. 

### Screenshots

image

### CODE REVIEW

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goals-coming-soon/dot-experiments-goals-coming-soon.component.scss [hunk 1]
```diff
@@ -0,0 +1,53 @@
+@use "variables" as *;
+
+:host {
+    display: flex;
+    background: $color-palette-gray-100;
+    flex-direction: column;
+    align-items: center;
+    border-radius: $border-radius-sm;
+    padding: 0.5rem;
```
Comment (zJaaal) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goals-coming-soon/dot-experiments-goals-coming-soon.component.scss:9]: We can use a global spacing variable for this

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goals-coming-soon/dot-experiments-goals-coming-soon.component.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    ::ng-deep {
+        .p-card {
+            border-radius: $border-radius-sm;
+
+            .p-card-body {
+                min-height: 9rem;
+
+                .p-card-title {
+                    font-size: 1rem;
```
Comment (zJaaal) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goals-coming-soon/dot-experiments-goals-coming-soon.component.scss:27]: We can use a global font size variable for this

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goals-coming-soon/dot-experiments-goals-coming-soon.component.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                min-height: 9rem;
+                padding: 0;
+                background-color: $color-palette-secondary-200;
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                border-radius: $border-radius-sm $border-radius-sm 0 0;
+
+                i {
+                    font-size: 1.5rem;
```
Comment (zJaaal) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goals-coming-soon/dot-experiments-goals-coming-soon.component.scss:41]: Same here

**Review by zJaaal — CHANGES_REQUESTED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goals-coming-soon/dot-experiments-goals-coming-soon.component.scss (hunk 1 above)
Comment (hmoreras) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-goals-coming-soon/dot-experiments-goals-coming-soon.component.scss]: @zJaaal feedback done

**Review by zJaaal — APPROVED** [review]

**Review by oidacra — APPROVED** [review]

**Pull request merged**